app/views/design-system/rsd-design-system/design-patterns/contact-details/index.html

```diff
@@ -0,0 +1,121 @@
+          codeSnippet.htmlCodeContactPerRowEdit, codeSnippet.nunjucksCodeContactPerRowEdit) }}
+
+          {{ sample.code('/design-system/rsd-design-system/design-patterns/contact-details/example-contact-per-row-editable.html', 'Contact details - contact per row (editable)', 'contact-per-row-editable-after',
+          codeSnippet.htmlCodeContactPerRowEditable, codeSnippet.nunjucksCodeContactPerRowEditable) }}
+
+          <h3>One contact per card</h3>
+  
+          <p>If grouping contacts together into a card would not work for your context, you can display each contact in a separate summary card.</p>
+        
+          <h4>Read only</h4>
+          If users just need to read contact details without editing them, use summary cards without change links.
+          {{ sample.code('/design-system/rsd-design-system/design-patterns/contact-details/example-contact-per-card-read-only.html', 'Contact details - contact per card (read-only)', 'contact-per-card-read-only',
+          codeSnippet.htmlCodeContactPerCardReadOnly, codeSnippet.nunjucksCodeContactPerCardReadOnly) }}
+
+          <h4>Editing contact details</h4>
+          <p>If users are allowed to edit the contact details, create an editing page and allow users to access it via a change link.</p>
+
+          <p>After users have edited the contact details, direct them back to the page which displays the edited details.</p>
+          {{ sample.code('/design-system/rsd-design-system/design-patterns/contact-details/example-contact-per-card-editable.html', 'Contact details - contact per card (editable)', 'contact-per-card-editable-before',
+          codeSnippet.htmlCodeContactPerCardEditable, codeSnippet.nunjucksCodeContactPerCardEditable) }}
+
+          {{ sample.code('/design-system/rsd-design-system/design-patterns/contact-details/example-contact-per-card-edit.html', 'Edit contact details (contact per card)', 'contact-per-card-edit',
+          codeSnippet.htmlCodeContactPerCardEdit, codeSnippet.nunjucksCodeContactPerCardEdit) }}
+
+          {{ sample.code('/design-system/rsd-design-system/design-patterns/contact-details/example-contact-per-card-editable.html', 'Contact details - contact per card (editable)', 'contact-per-card-editable-after',
+          codeSnippet.htmlCodeContactPerCardEditable, codeSnippet.nunjucksCodeContactPerCardEditable) }}
+        
+        <h2>Research on this pattern</h2>
+        <p>We have not done research on this design pattern, but the pattern is being used in <a href="https://design-histories.education.gov.uk/prepare-conversions-transfers">Prepare conversions and transfers</a> and <a href="https://design-histories.education.gov.uk/complete-conversions-transfers-and-changes">Complete conversions, transfers and changes</a> to meet these user needs from our <a href="https://educationgovuk.sharepoint.com/sites/ServiceDeliveryDirectorate/Lists/RSD%20crossprogramme%20user%20needs/AllItems.aspx?env=WebViewList">user needs database</a>:</p>
```

Comment:
See previous comments about internal links

Reply:
This part is now:

```
<p>We have not done research on this design pattern, but the pattern is being used in <a href="https://design-histories.education.gov.uk/prepare-conversions-transfers">Prepare conversions and transfers</a> and <a href="https://design-histories.education.gov.uk/complete-conversions-transfers-and-changes">Complete conversions, transfers and changes</a> to meet these user needs from our <a href="https://educationgovuk.sharepoint.com/sites/ServiceDeliveryDirectorate/Lists/RSD%20crossprogramme%20user%20needs/AllItems.aspx?env=WebViewList">user needs database (opens in DfE Microsoft List)</a>:</p>```